engine/src/build/config/ios/ios_sdk.gni

```diff
@@ -66,3 +74,23 @@ if (ios_sdk_path == "") {
     ios_sdk_path = ios_device_sdk_path
   }
 }
+
+if (ios_toolchain_path == "") {
```

Comment:
`et` runs `gn` unconditionally on every build. Running python scripts during `gn` is very expensive, and gets annoying when iterating on small incremental builds. To avoid that, `gn` should populate these variables for local builds. We do this for the SDK paths here: https://github.com/flutter/flutter/blob/127faefdf2c87260e36e3f158bc161b87161cfe9/engine/src/flutter/tools/gn#L264.

Reply:
Done. And extracted it as apple_host_toolchain_path in a common file shared by both iOS and macOS, since it's part of the host toolchain.